src/libstd/primitive_docs.rs

```diff
@@ -592,6 +592,11 @@ mod prim_array {}
 /// x[1] = 7;
 /// assert_eq!(x, &[1, 7, 3]);
 /// ```
+///
+/// It is also possible to put a slice in a box, when all you have are a pointer
+/// and a length: `Box::from_raw(slice::from_raw_parts_mut(ptr, len) as *mut [u8])`.
```

Comment:
I’m not sure if there’s any issue with this proposed code, but `Box::from_raw(ptr::slice_from_raw_parts_mut(ptr, len))` avoids going through `&mut [_]`, we could recommend that instead.